cron/poll_pull_requests.py

```diff
@@ -76,30 +76,36 @@ def poll_pull_requests(api):
             threshold = gh.voting.get_approval_threshold(api, settings.URN)
             is_approved = vote_total >= threshold and meritocracy_satisfied
 
+            seconds_since_updated = gh.prs.seconds_since_updated(api, settings.URN, pr)
+
+            voting_window = base_voting_window
             # the PR is mitigated or the threshold is not reached ?
             if variance >= threshold or not is_approved:
                 voting_window = gh.voting.get_extended_voting_window(api, settings.URN)
```

Comment:
Previously, this would update the global `voting_window`, as Python doesn't have any scope beyond functions and classes. That meant any PR processed after one with the extended voting window would also get the extended voting window.